### ISSUE TITLE
AttributeAndTargetSpacing: false positive with comments used

### ISSUE THREAD

**Issue by janedbal** [slevomat/coding-standard#1462]
Following code reports violation on line with UniqueConstraint: `Expected 0 blank lines between attribute and its target, both are on same line.`. No issue when comment is removed.

Using version 8.6.2

```php
#[UniqueConstraint] // comment
#[Entity]
class MyEntity {}
```

**Comment by kukulich** [comment]
Thank you for your report.

Fixed in https://github.com/slevomat/coding-standard/commit/82370d1fa3234eba4b731e8a577c83c56f0a8640

**Comment by janedbal** [comment]
Thank you!
